lambdas/file_access_patterns/lambda_for_fileUpload/index.js

```diff
@@ -17,44 +17,46 @@ function generateTimestamp() {
   return timestamp;
 }
 
-exports.uploadFile = async (event) => {
+exports.handler = async (event) => {
   const reqBody = event.body;
   var d = new Date();
   const timestamp = generateTimestamp();
   const file = {
-    PK: `USER#${event.path.u_id}`,
+    PK: `USER#${event.path.userId}`,
     SK: `FILE#${timestamp}`,
     LSI_SK: reqBody.f_name,
     size: reqBody.f_size,
     hash: reqBody.f_hash,
-    type:reqBody.f_type
+    type: reqBody.f_type,
   };
 
-  try{
-    await dynamo.put({
-      TableName:"V-Transfer",
-      Item: file
-    }).promise()
+  try {
+    await dynamo
+      .put({
+        TableName: "V-Transfer",
+        Item: file,
+      })
+      .promise();
+  } catch (err) {
+    return response(err.statusCode, err.message);
   }
-  catch(err){
-    return response(err.statusCode,err.message)
-  }
-  
-  try{
-    await dynamo.update({
-      TableName: "V-Transfer",
-      Key: {
-        PK: `USER#${event.path.u_id}`,
-        SK: `METADATA`,  //token is password hash
-      },
-      UpdateExpression: "add storage_used :file_size",
-      ExpressionAttributeValues: {
-        ":file_size": reqBody.f_size,
-      },
-    }).promise()
-    return response(201,"File upload success")
-  }
-  catch(err){
-    return response(err.statusCode,err.message)
+
+  try {
+    await dynamo
+      .update({
+        TableName: "V-Transfer",
+        Key: {
+          PK: `USER#${event.path.userId}`,
+          SK: `METADATA`, //token is password hash
```

Comment:
what is this `//token is password hash` comment for??

Reply:
Ignore it. Will remove it.

Reply:
fix it